### ISSUE TITLE
Accessibility Improvement: "Form elements do not have associated labels"

### ISSUE THREAD

**Issue by xavierloeraflores** [kevincarringtonstory/word-chain#10]
Form input element is missing a label lowering the accessibility score. 

<img width="870" alt="image" src="https://github.com/user-attachments/assets/0aa523bc-11e6-4ce5-b6a7-b142e3f305b4">
